ExampleGenerator/Series/StemSeriesExamples.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace ExampleGenerator
+{
+	using System;
+	using OxyPlot;
+	using OxyPlot.Axes;
+	using OxyPlot.Series;
+
+	public class StemSeriesExamples
+	{
+
+		[Export(@"Series\StemSeries")]
+		public static PlotModel StemSeries()
+		{
+			var model = new PlotModel{ Title = "Trigonometric functions" };
+
+			var start = -Math.PI;
+			var end = Math.PI;
+			var step = 0.1;
+			int steps = (int)((Math.Abs(start) + Math.Abs(end)) / step);
+
+			//generate points for functions
+			var sinData = new DataPoint[steps];
+			var cosData = new DataPoint[steps];
+			for(int i = 0; i < steps; ++i) {
+				var x = (start + step * i);
+				sinData[i] = new DataPoint(x, 0.5 * Math.Sin(x));
+
+				cosData[i] = new DataPoint(x, Math.Cos(x));
+			}
+
+			//0.5 * sin(x)
+			var sinStemSeries = new StemSeries
+			{
+				MarkerStroke = OxyColors.Green,
+				MarkerType = MarkerType.Circle
+			};
+			sinStemSeries.Points.AddRange(sinData);
+
+			//cos(x)
+			var cosStemSeries = new StemSeries
+			{
+				MarkerStroke = OxyColors.DarkOrange,
+				MarkerType = MarkerType.Circle
+			};
+			cosStemSeries.Points.AddRange(cosData);
+
+			model.Series.Add(sinStemSeries);
+			model.Series.Add(cosStemSeries);
+
+			return model;
+		}
+
+
+		[Export(@"Series\StemSeries_advanced")]
+		public static PlotModel StemSeries_advanced()
```

Comment:
No new functionality demonstrated, so I don't think this is a good example. Even if it is a fun function :-)
I think focus should be on the StemSeries functionality (e.g. the `Base` property), and keep the other code to a minimum.